conftest.py

```diff
@@ -0,0 +1,175 @@
+"""
+Title:   conftest.py
+Purpose: Contains hooks and fixtures for yt testing
+Notes:
+    1.) https://docs.pytest.org/en/latest/example/simple.html
+    2.) https://docs.pytest.org/en/latest/historical-notes.html#string-conditions
+    3.) https://docs.pytest.org/en/latest/skipping.html#skipif
+    4.) https://docs.pytest.org/en/latest/reference.html
+"""
+import os
+import shutil
+import tempfile
+
+import pytest
+import yaml
+
+from yt.config import ytcfg
+from yt.utilities.answer_testing import utils
+
+
+# Global variables can be added to the pytest namespace
+pytest.answer_files = {}
+
+# List of answer files
+answer_file_list = 'tests/tests.yaml'
+answer_dir = os.path.join(ytcfg.get('yt', 'test_data_dir'), 'answers')
+
+
+def pytest_addoption(parser):
```

Comment:
I just tried it in a minimum working example and it turns out that it can't. It complains about pytest_add_option being an unknown hook. I could probably register it as a hook or some such thing, if you'd like?

Reply:
nah it's fine